_posts/digests/2019-09-22-32.md

```diff
@@ -0,0 +1,128 @@
+
+* [New AI Face Anonymization Model Protects Privacy](https://medium.com/syncedreview/new-ai-face-anonymization-model-protects-privacy-a7d2d293fa8a) - Was George Orwell right, is Big Brother watching us? Undoubtedly many are alarmed by the ever-increasing level of computer-driven surveillance, particularly involving facial recognition technologies.
+
+* [AI Decoded: Evaluating Artificial Intelligence Startups](https://www.forbes.com/sites/jilliandonfro/2019/09/17/meritech-capital-konstantine-buhler-ai-decoded-evaluating-artificial-intelligence-startups/) - Since the term was coined in 1956, Artificial Intelligence has endured a lifetime of misunderstanding. The root of the problem lies in the interpretation of the word 'intelligence.'
+
+* [AI learned to use tools and cooperate after 380 million games of hide and seek](https://www.technologyreview.com/s/614325/open-ai-algorithms-learned-tool-use-and-cooperation-after-hide-and-seek-games/) - In the early days of life on Earth, biological organisms were exceedingly simple. They were microscopic unicellular creatures with little to no ability to coordinate.
+
+* [AI can gauge the risk of dying from heart conditions](https://www.engadget.com/2019/09/15/mit-machine-learning-cardiovascular-death-risk/) - AI's ability to predict threats to your health could soon include deadly heart conditions. Researchers at MIT's CSAIL have developed a machine learning system, RiskCardio, that can estimate the risk of death due to cardiovascular issues that block or reduce blood flow.
+
+* [FarmWise and its weed-pulling agribot harvest $14.5M in funding](https://techcrunch.com/2019/09/17/farmwise-and-its-weed-pulling-agribot-harvest-14-5m-in-funding/) - Automating agriculture is a complex proposition given the number and variety of tasks involved, but a number of robotics and autonomy companies are giving it their best shot. FarmWise seems to have impressed someone--it just raised $14.
+
+* [Amazon Releases Data Set of Annotated Conversations to Aid Development of Socialbots](https://developer.amazon.com/blogs/alexa/post/885ec615-314f-425f-a396-5bcffd33dd76/amazon-releases-data-set-of-annotated-conversations-to-aid-development-of-socialbots) - Amazon announces the public release of the Topical Chat Dataset, a text-based collection of more than 235,000 utterances (over 4,700,000 words) that will help support high-quality, repeatable research in the field of dialogue systems.
+
+* [AI learns to defy the laws of physics to win at hide-and-seek](https://www.newscientist.com/article/2216410-ai-learns-to-defy-the-laws-of-physics-to-win-at-hide-and-seek/) - Never play games with a bot--it will find a way to cheat if they can. A team from OpenAI, an artificial intelligence lab in San Francisco co-founded by Elon Musk, has developed artificially intelligent bots that taught themselves to cooperate by playing hide-and-seek.

```

Comment:
remove b/c duplicate of OpenAI's article (currently in the expert discussion section)